Measurements/README.md

```diff
@@ -7,4 +7,4 @@ The measurements kata covers the following topics:
 Variations of quantum state discrimination tasks are covered in the paper ["Quantum State Discrimination"](https://arxiv.org/pdf/quant-ph/0010114.pdf).
 * Task 2.1 is an example of hypothesis testing for two pure states.
 * Task 2.2 is an example of unambiguous state discrimination. See also the paper ["Unambiguous quantum measurement of nonorthogonal states"](https://www.researchgate.net/publication/13375059_Unambiguous_quantum_measurement_of_nonorthogonal_states)
-  for further information and hints about how to implement the unambiguous measurements required for this task. 
+  for further information and hints about how to implement the unambiguous measurements required for these tasks. 
```

Comment:
I don't think you need this change - only task 2.2 deals with unambiguous state discrimination, that's singular. Could you remove this file from the PR?

Reply:
I went on and undid this change, so that I can merge.